### ISSUE TITLE
Docker install issue

### ISSUE THREAD

**Issue by midnight38** [tinode/chat#73]
Hi. I have a problem installing tinode server on windows 10 using docker.
On point 5 - $ docker run --rm --name tinode-init-db --network tinode-net tinode-init-db in console i've got this:

standard_init_linux.go:195: exec user process caused "no such file or directory"

On stackoverflow i don't found decision for solving this problem. Do you have any ideas?

**Comment by or-else** [comment]
Unfortunately I don't have access to Windows so I can't reproduce. 

Is this the only message in the console?

Here is a discussion of something very similar https://github.com/docker/labs/issues/215

**Comment by or-else** [comment]
Here is another discussion https://forums.docker.com/t/standard-init-linux-go-175-exec-user-process-caused-no-such-file/20025/3 claiming that the problem is due to CR-LF line breaks on windows vs CR on Linux in entrypoint.sh. Clearly it's a Docker bug. 

Do you use the latest Docker?

**Comment by or-else** [comment]
Here is a possible solution: https://stackoverflow.com/questions/38905135/why-wont-my-docker-entrypoint-sh-execute (the second answer with 32 votes):

> The solution for me was to disable git's automatic conversion: git config --global core.autocrlf input Then check out the repository again and rebuild

**Comment by or-else** [comment]
Can you try the docker file from `devel`? It should not have this issue.

**Comment by midnight38** [comment]
Thanks! The solution with disabling git's automatic conversion helped to solve the problem. 
But now i have another issue with runing tinode-srv.
I think it's a problem with ports on Windows.

Is there a reason that you setting the port mapping -p 6060:18080 ?

**Comment by midnight38** [comment]
If someone will need this info for installing tinode on windows there is a solution.

After step 6 you need to install httpd:
$docker pull httpd

On step 7 write:
$docker run -d --name tinode-srv --network tinode-net -p 80:80 httpd

Then test the installation by pointing your browser to http://localhost

If there will be an error on step 7 you will need to stop all the containers:
$docker ps -a -q | ForEach { docker stop $_ }

Then restart the Docker and repeat step 7.

Articler: https://github.com/docker/for-win/issues/204
https://github.com/docker/for-win/issues/573

**Comment by or-else** [comment]
> Is there a reason that you setting the port mapping -p 6060:18080 ?

You mean as opposite to just port 80? On linux and Mac ports below 1024 require root privileges and as such it unnecessarily complicates things.

**Comment by or-else** [comment]
BTW, would you be willing to create a pull request with instructions for Windows? Thanks!
